faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
```

Comment:
On Wed, Mar 18, 2015 at 06:08:11PM -0700, Greg Wilson wrote:
> 1.  +1 on querying Amy for URLs of upcoming workshops.

I was suggesting querying amy for *all* the data, and not requiring a
workshop URL.

> 2.  We should fetch the actual info of those workshops from their
> websites (since instructors, rooms, etc. can change at the last
> minute).

I'm ok with this if the workshop has a URL, but I'd rather have
workshop URLs continue to be optional.  I'm not sure how this works
currently (I'd expect you'd need to rebuild swcarpentry/site to get
the new values anyway?).  My favorite approach here would be to have
the swcarpentry/site listing hitting a public, CORS-enabled amy
endpoint to get this data dynamically, and then have amy periodically
poll pending workshops with URLs to pull in new data from the workshop
pages.  Once we give instructors access to amy to update their pending
event data directly, we can drop the polling.

> 3.  We need to archive info about past workshops in the `site` repo
> - can't be sure their repos will still be there in a year.

I'm suggesting we store all of this in amy, so that shouldn't be a
problem.